tidy3d/plugins/invdes/optimizer.py

```diff
@@ -65,6 +66,12 @@ class AbstractOptimizer(InvdesBaseModel, abc.ABC):
         "last computed state of these variables.",
     )
 
+    parameter_spec: typing.Optional[ParameterSpecType] = pd.Field(
+        None,
```

Comment:
should the default just be a `UniformParameterSpec()`?

Reply:
if we don't support `params0` then yes, otherwise we'd have to let `params0` take precedence over this if defined

Reply:
no I guess this is fine, especially if we support passing custom parameters via `CustomParameterSpec`